src/uct/ib/ud/accel/ud_mlx5.c

```diff
@@ -638,6 +638,17 @@ static void uct_ud_mlx5_iface_event_cq(uct_ib_iface_t *ib_iface,
 
 static void UCS_CLASS_DELETE_FUNC_NAME(uct_ud_mlx5_iface_t)(uct_iface_t*);
 
+static void uct_ud_mlx5_iface_handle_failure(uct_ib_iface_t *iface, void *arg,
+                                             ucs_status_t status)
+{
+    if (status != UCS_ERR_ENDPOINT_TIMEOUT) {
+        /* Local side failure - treat as fatal */
+        uct_ib_mlx5_completion_with_err(iface, arg, UCS_LOG_LEVEL_FATAL);
+    }
+
```

Comment:
do we need 'else' here? or at least add ucs_fatal/assert that uct_ib_mlx5_completion_with_err() never returns. Since we cannot enter uct_ud_iface_handle_failure(), it will segfault

Reply:
`uct_ib_mlx5_completion_with_err` calls `ucs_log` with `UCS_LOG_LEVEL_FATAL` for all cases excepting MLX5_CQE_SYNDROME_WR_FLUSH_ERR, so we won't return from `uct_ib_mlx5_completion_with_err` after first error, right?

Reply:
so need to add 'else' here, don't want to rely on the fact that FLUSH_ERROR is never first

Reply:
done